controls.py

```diff
@@ -0,0 +1,80 @@
+import pygame
+import sys
+
+
+class Controls:
+    def __init__(self, comms, exit_program):
+        self.comms = comms
+        self.exit_program = exit_program
+        
+        self.running = True
+        self.SCREEN_COLOR = (0, 0, 0)
```

Comment:
consider adding parameters to the constructor with `SCREEN_COLOR` and `SCREEN_SIZE`, using the current values as a default instead.

Reply:
I will hold of on this until I talk to Liam.

Reply:
On second thoughts, I'll make it an optional parameter.